backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewCreateService.java

```diff
@@ -360,19 +363,19 @@ public List<Feature> getInheritedCompartmentItems(Type type, String eReferenceNa
         EStructuralFeature eStructuralFeature = type.eClass().getEStructuralFeature(eReferenceName);
         if (eStructuralFeature instanceof EReference eReference) {
             type.getInheritedFeature().stream()
-                    .filter(feature -> new InheritedCompartmentItemFilterSwitch(eReference).doSwitch(feature))
-                    .forEach(inheritedElements::add);
+            .filter(feature -> new InheritedCompartmentItemFilterSwitch(eReference).doSwitch(feature))
```

Comment:
Please restore original formatting, and do it everywhere it is necessary in this PR